modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1972,6 +1976,77 @@ public void testGetObjectEntryRelatedObjectEntriesWithAccountEntryRestricted()
 			objectRelationship2.getObjectRelationshipId());
 	}
 
+	@Test
+	public void testPartialUpdateObjectEntry() throws Exception {
+		ObjectEntry objectEntry = _defaultObjectEntryManager.addObjectEntry(
+			_dtoConverterContext, _objectDefinition2,
+			new ObjectEntry() {
+				{
+					properties = HashMapBuilder.<String, Object>put(
+						"dateObjectFieldName", "2023-08-10"
+					).put(
+						"decimalObjectFieldName", 2.7
+					).put(
+						"integerObjectFieldName", 50
+					).put(
+						"longIntegerObjectFieldName", 30000L
+					).put(
+						"precisionDecimalObjectFieldName",
+						new BigDecimal(
+							0.9876543217654321, MathContext.DECIMAL64)
+					).put(
+						"richTextObjectFieldName",
+						StringBundler.concat(
+							"<i>", RandomTestUtil.randomString(), "</i>")
+					).put(
+						"textObjectFieldName", RandomTestUtil.randomString()
+					).build();
+				}
+			},
+			ObjectDefinitionConstants.SCOPE_COMPANY);
```

Comment:
@pedro-oliveira446 use random values for  dateObjectFieldName, decimalObjectFieldName, integerObjectFieldName, longIntegerObjectFieldName and precisionDecimalObjectFieldName as well

Reply:
```java
SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd");

return simpleDateFormat.format(RandomTestUtil.nextDate());
```